netbox/templates/base/layout.html

```diff
@@ -179,9 +179,9 @@ <h1 class="navbar-brand navbar-brand-autodark">
             {# /Footer links #}
 
             {# Footer text #}
-            <ul class="list-inline list-inline-dots mb-0" id="footer-stamp" hx-swap-oob="true">
+            <ul class="list-inline list-inline-dots fs-5 mb-0" id="footer-stamp" hx-swap-oob="true">
               <li class="list-inline-item">
-                {% annotated_now %} {% now 'T' %}
+                {% now 'Y-m-d H:i:s T' %}
```

Comment:
Will that format be a 'standard' potentially used elsewhere in the future? If so we should put it into a filter or constant.